1/src/Shape.ts

```diff
@@ -0,0 +1,43 @@
+import {Point} from "./point";
```

Comment:
You should have consistency in your code-base. With tools like eslint and prettier you will not have this kind of lint issues. Everywhere else there are spaces  after and before braces `import { Point }`